created user order queue

ibdthor opened the pull request:
created single order queue for user mobile site

ibdthor committed:
created user order queue

created single order queue for user mobile site

joshuafcole commented on food-truck.eve.md:
````diff
@@ -84,7 +84,7 @@ commit
 
 display the nav button, which is unconditional
-```
+```eve disabled
````
Is this intentionally disabled?

joshuafcole commented on food-truck.eve.md:
````diff
@@ -19,14 +19,14 @@ we can add the additional imagery to the page if provided
 Draw the homepage.
 
-```
+```eve disabled
````
Why did you disable the homepage?

joshuafcole commented on food-truck.eve.md:
````diff
@@ -156,148 +156,104 @@ commit
-- Clicking pay button takes you to the Stripe payment page/overlay
+```
+search
+  order = [#order #my-order number items status]
````
You should search to make sure the app is on the current page at the top of this block. Look at the homepage for an example.

joshuafcole commented on food-truck.eve.md:
```diff
@@ -156,148 +156,104 @@ commit
+  flex-grow: 10;
+}
+
+.back {
```
We should be a little more disciplined about our classnames here, since anything else in the app that wants to use a name like `.back` will end up picking these styles up. In general it's good practice to have all of your selectors have two parts -- the classname of the container (in this case, maybe the page) and the classname of the thing to style (in this case, the back button). E.g. `.orders-page .back-btn { ...`

joshuafcole commented on food-truck.eve.md:
````diff
@@ -156,148 +156,104 @@ commit
+  padding-left: 20px;
+  padding-top: 20px;
+}
+```
````
This is just a BS editor bug and not your fault, but make sure you *always* have an extra newline at the end of these blocks. The way the code currently works just sticks every CSS block directly together, which messes up the following blocks.

joshuafcole commented on food-truck.eve.md:
```diff
@@ -375,45 +331,38 @@ commit
    [#menu name:"Bacon Swiss Burger"
     image:"assets/burger.jpg"
-    description:"A half pound burger made from Niman Ranch beef with melted Swiss, thick cut bacon, and housemade aioli and ketchup."
+    description:"A half pound Niman Ranch burger with melted Swiss, thick cut bacon, and housemade aioli and ketchup."
```
Thanks, these changes will make the menu look a lot nicer I think.

joshuafcole commented on food-truck.eve.md:
````diff
@@ -570,6 +519,61 @@ bind @browser
+  border: 1px solid #f0f0f0;
+  color: #000000;
+}
+```
````
Needs a newline

joshuafcole commented on npm-debug.log:
```diff
@@ -0,0 +1,110 @@
+0 info it worked if it ends with ok
```
Don't want to commit this file, let's get @cmontella's help to help you remove it from the PR (I have no idea how the GitHub app works, only git).

ibdthor committed:
fixes + view control

fixed disabled blocks, added page search and view control to make sure
only chosen page is drawn, cleaned up CSS class names for specificity,
added extra lines in CSS for editor bug, and deleted whatever that
weird npm-debug.log file was

joshuafcole commented:
🤔 💭 🇰🇳
